app/models/admin_section.rb

```diff
@@ -10,6 +10,10 @@ def self.permissions
     end
   end
 
+  def self.permissions_for_account(account)
+    account.provider_can_use?(:policies_registry) ? permissions : (permissions - %i[policies_registry])
```

Comment:
hmmm, just `policies_registry` is the only permission under rolling update for accounts?

Reply:
So far, yes, because it was decided to introduce this custom policies endpoints as a new permission section at the same time we wanted to put the creation of custom policies in Saas under rolling update. So there's no point in showing the permission to user that cannot use them. It would be a no-op.

This code is supposed to be short-lived anyway. As soon as we can rollout `policy_registry` to all customers that already have `policies`, then we can go back to showing all the permissions to all customers I guess.